### PR TITLE
Nullwave Vibrato now works as intended (fix and balance)

### PR DESCRIPTION
**Pull request by tralezab**
## About The Pull Request

As it is right now:
hearing any of it applies 2 minutes of antimagic, which doesn't refresh even if you keep listening to the song.
After this change:
hearing the song applies 10 seconds of antimagic, which stacks. So one process of the song gives ~15 seconds since about 5 seconds pass before reapplication, two gives 20 seconds, three gives 25 seconds, and so on and so forth.

## Why It's Good For The Game

Nullwave Vibrato code wasn't doing what it was supposed to, and for some reason was set to stack 2 minutes instead of 10 seconds, which would have quickly led to full round antimagic. It's already very strong as an AOE tool that hard counters multiple antagonists, so if it was JUST fixed it would have been even crazier than it was now.

## Changelog

:cl:
fix: Nullwave Vibrato now works as intended, stacking antimagic duration
balance: Nullwave Vibrato stacks 10 seconds of antimagic, every few moments you listen to it.
/:cl:

### CODE REVIEW

File: code/modules/religion/festival/instrument_rites.dm [hunk 1]
```diff
@@ -82,7 +82,7 @@
 	for(var/mob/living/listener in song_datum.hearing_mobs)
 		if(listener.can_block_magic(MAGIC_RESISTANCE_HOLY, charge_cost = 1))
 			continue
-		listener.apply_status_effect(/datum/status_effect/antimagic, 2 MINUTES)
```
Comment (tralezab) [inline code/modules/religion/festival/instrument_rites.dm:85]: i don't even... i don't know, okay?

**Comment by timothymtorres** [comment]
Very cool effect to have antimagic gained from a song.  I was thinking of expanding it to other items later at some point but this is pretty neat!

**Comment by TemporalOroboros** [comment]
By stacking do you mean refreshing the timer or resetting the timer and adding an additional ten seconds? Ie: is 25 seconds at three stacks measured from the first stack or the third stack?

**Comment by tralezab** [comment]
> By stacking do you mean refreshing the timer or resetting the timer and adding an additional ten seconds? Ie: is 25 seconds at three stacks measured from the first stack or the third stack?

It doesn't reset the timer, but it extends it by 10 seconds. So it's still counting down at the point it was at, but with 10 more seconds. Since it takes around 5 seconds per application of antimagic, you'd have 25 seconds from 3 applications because 15 seconds would have passed

**Comment by ATH1909** [comment]
did you check to see if this prevents affected people from casting spells as well? if so, this is a pretty massive "fuck you, die now" to cultists, heretics, etc.

**Comment by timothymtorres** [comment]
The antimagic trait only blocks receiving magic not casting.

File: code/datums/status_effects/song_effects.dm [hunk 2]
```diff
@@ -0,0 +1,32 @@
+///Type of status effect applied by music played by the festival sect. Stacks upon itself, and removes all other song subtypes other than itself.
+/datum/status_effect/song
+	id = "pleaseno"
```
Comment (MrMelbert) [inline code/datums/status_effects/song_effects.dm:3]:
```suggestion
	id = "pleaseno"
	alert_type = null
```

**Pull request merged**